lib/mongoidable/rspec/controller_matchers.rb

```diff
@@ -0,0 +1,151 @@
+require "rspec/expectations"
+
+module Mongoidable
+  module RSpec
+    module ControllerMatchers
+      extend ::RSpec::Matchers::DSL
+
+      class ControllerHelper < SimpleDelegator
+        def initialize(obj, controller, authorize_pairs)
+          super(obj)
+
+          @controller = controller
+          @authorize_pairs = authorize_pairs
+        end
+
+        def error_message
+          "The #{controller.class.name} did not authenticate the action #{action} against the specified object."
+        end
+
+        def process_action
+          setup_controller
+
+          matchers = authorization_matcher
+          controller._run_process_action_callbacks
+
+          loaded_matcher
```

Comment:
I'm pretty sure that this is old code of mine that I had abandoned, but apparently didn't take out - especially since I can't find it being used any more.  Sorry.